mvpa2/base/config.py

```diff
@@ -225,8 +225,7 @@ def get_as_dtype(self, section, option, dtype, default=None):
             return default
         try:
             return SafeConfigParser._get(self, section, dtype, option)
-        except ValueError, e:
+        except ValueError as e:
             # provide somewhat descriptive error
-            raise ValueError, \
-                  "Failed to obtain value from configuration for %s.%s. " \
-                  "Original exception was: %s" % (section, option, e)
+            raise ValueError("Failed to obtain value from configuration for %s.%s." \
```

Comment:
Just FYI: no trailing `\` would longer be necessary here since now both strings are within `()`